### ISSUE TITLE
[fs-bq-import-collection] Overriding destination BigQuery project ID

### ISSUE THREAD

**Issue by manikant-prasad** [firebase/extensions#969]
### Extension name

This feature request is for extension: `fs-bq-import-collection`

### What feature would you like to see?

The extension for changelog allows us to override the destination BigQuery project ID which is not the case with the one-time import script. It never prompts for the BigQuery Project ID and the same project ID (as Firestore) is being used as the destination of the BigQuery instance. 

It doesn't even allow passing the BigQuery project ID along with the dataset ID to override the default project ID. 

### How would you use it?

Our current changelog exports are set to export data to a BiggQuery instance outside the project containing the Firestore instance.  Need a way to be able to create these one-time dumps on the same tables (different BigQuery project ID).

**Comment by yamankatby** [comment]
Blocked by another PR, waiting for #979 to be merged.

**Comment by jauntybrain** [comment]
This should be resolved with #1800.

**Comment by pr-Mais** [comment]
Feel free to open a new issue if the latest version doesn't solve the issue.
